Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs

```diff
@@ -12,13 +14,13 @@ public class GetTagByTitleHandler : IRequestHandler<GetTagByTitleQuery, Result<T
 {
     private readonly IMapper _mapper;
     private readonly IRepositoryWrapper _repositoryWrapper;
-    private readonly ILoggerService _logger;
+    private readonly IStringLocalizer<CannotFindSharedResource> _stringLocalizerCannotFind;
 
-    public GetTagByTitleHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper, ILoggerService logger)
```

Comment:
Here you can see the logger replacement.

Reply:
done